python/huggingfaceserver/huggingfaceserver/__main__.py

```diff
@@ -68,6 +68,9 @@ def list_of_strings(arg):
 parser.add_argument(
     "--return_token_type_ids", action="store_true", help="Return token type ids"
 )
+parser.add_argument(
+    "--enable_triton", action="store_true", help="Use triton as the runtime"
```

Comment:
We have introduced the new argument `backend` for huggingfaceserver, so we can add `triton` as another backend in addition to [vllm, huggingface]